assets/src/edit-story/elements/video/output.js

```diff
@@ -42,11 +42,23 @@ function VideoOutput({ element, box }) {
     loop: loop ? 'loop' : undefined,
   };
 
+  const tracks = defaultForUndefined(element.tracks, resource.tracks);
+
   // crossorigin='anonymous' is required to play videos from other domains.
   return (
     <MediaOutput element={element} box={box}>
       <amp-video {...props} id={`el-${element.id}-media`}>
         <source {...sourceProps} />
+        {tracks &&
+          tracks.map(({ srclang, label, kind, track, id }) => (
+            <track
+              srcLang={srclang}
+              label={label}
+              kind={kind}
+              track={track}
+              key={id}
+            />
```

Comment:
```suggestion
          tracks.map(({ id, ...rest }) => (
            <track
              key={id}
              ...rest
            />
```

Reply:
There are lots of params here that are not needed, like id and trackId. This is why I did not do this.